CHANGELOG for v1.x.x.md

```diff
@@ -349,8 +263,6 @@
 
 * #3222 [fixed] - UI issue in event ticket booking special price date field
 
-* #3227 [fixed] - Guest coupon code cannot be used for general users
-
 * #3228 [fixed] - product videos in e-commerce 
```

Comment:
This is also a general question / request

Reply:
Done